admin/views/tabs/dashboard/general.php

```diff
@@ -66,7 +66,7 @@
 </p>
 
 <p>
-	<a onclick="if ( !confirm( '<?php esc_html_e( 'Are you sure you want to reset your SEO settings?', 'wordpress-seo' ); ?>' ) ) return false;"
+	<a onclick="if ( !confirm( '<?php echo esc_attr( wp_json_encode( __( 'Are you sure you want to reset your SEO settings?', 'wordpress-seo' ) ) ); ?>' ) ) return false;"
```

Comment:
As `wp_json_encode` already provides the string with quotes, the surrounding quotes `'` are redundant and make the message have double-quotes internally.

Reply:
Thanks. Updated. Feel free to squash on merge.

Looks like the return from `wp_json_encode()` always uses double quotes, so I've changed the quotes for the `onclick` to single quotes.

This will still need additional testing as the `esc_attr()` is supposed to encode quotes, so I'm no doubting this will work as it should.